common/test/acceptance/tests/discussion/test_discussion_management.py

```diff
@@ -271,3 +288,164 @@ def test_verify_that_deselecting_the_final_child_deselects_category(self):
+            self.discussion_management_page.division_scheme_visible(
+                self.discussion_management_page.ENROLLMENT_TRACK_SCHEME
+            )
+        )
+
+    def test_enrollment_track_scheme_persisted(self):
+        """
+        Test that the enrollment track scheme is visible if there are multiple enrollment tracks,
+        and that the selection can be persisted.
+
+        Also verifies that the cohort division scheme is not presented if cohorts are disabled and cohorts
+        are not the selected division scheme.
+        """
+        self.add_enrollment_modes(['audit', 'verified'])
+        self.assertTrue(
+            self.discussion_management_page.division_scheme_visible(
+                self.discussion_management_page.ENROLLMENT_TRACK_SCHEME
+            )
+        )
+        # And the cohort scheme is initially visible because it is selected (and cohorts are enabled).
+        self.assertTrue(
+            self.discussion_management_page.division_scheme_visible(self.discussion_management_page.COHORT_SCHEME)
+        )
+
+        self.discussion_management_page.select_division_scheme(self.discussion_management_page.ENROLLMENT_TRACK_SCHEME)
+        self.verify_save_confirmation_message(self.scheme_key)
+        self.check_discussion_topic_visibility(visible=True)
+
+        # Also disable cohorts so we can verify that the cohort scheme choice goes away.
+        self.disable_cohorting(self.course_fixture)
+
+        self.reload_page(topics_visible=True)
+        self.assertEqual(
+            self.discussion_management_page.ENROLLMENT_TRACK_SCHEME,
+            self.discussion_management_page.get_selected_scheme()
+        )
+        # Verify that the cohort scheme is no longer visible as cohorts are disabled.
+        self.assertFalse(
+            self.discussion_management_page.division_scheme_visible(self.discussion_management_page.COHORT_SCHEME)
+        )
+
+    def test_enrollment_track_discussion_visibility_label(self):
+        """
+        If enrollment tracks are the division scheme, verifies that discussion visibility labels
+        correctly render.
+
+        Note that there are similar tests for cohorts in test_cohorts.py.
+        """
+        def refresh_thread_page():
+            self.browser.refresh()
+            self.thread_page.wait_for_page()
+
+        # Make moderator for viewing all groups in discussions.
+        AutoAuthPage(self.browser, course_id=self.course_id, roles="Moderator", staff=True).visit()
+
+        self.add_enrollment_modes(['audit', 'verified'])
+        self.discussion_management_page.select_division_scheme(self.discussion_management_page.ENROLLMENT_TRACK_SCHEME)
+        self.verify_save_confirmation_message(self.scheme_key)
+        # Set "always divide" as the thread we will be creating will be an inline thread,
+        # and this way the thread does not need to be explicitly divided.
+        self.enable_always_divide_inline_discussions(self.course_fixture)
+
```

Comment:
Uck, this extra blank line caused a quality issue. I will fix when addressing feedback.